src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/jpa/repository/PathwayStatusRepository.kt

```diff
@@ -9,4 +10,7 @@ import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.entity.Pris
 @Repository
 interface PathwayStatusRepository : JpaRepository<PathwayStatusEntity, Long> {
   fun findByPathwayAndPrisoner(pathway: PathwayEntity, prisoner: PrisonerEntity): PathwayStatusEntity?
+
+  @Query("select p.nomsId from PathwayStatusEntity ps, PrisonerEntity p where ps.status.id != :status and p.prisonId = :prisonId and p.id = ps.prisoner.id group by p.nomsId having count(ps) > 0")
+  fun findInusePrisonersByPrisonIdAndStatus(prisonId: String, status: Long): List<String>
```

Comment:
Can this be `InUse` rather than `Inuse`?

Reply:
Done